README.md

```diff
@@ -18,4 +18,7 @@ brew install mongodb-community
 3. Start mongodb via: `brew services run mongodb-community` or stop via `brew services run mongodb-community`.
 
 ### For window users:
-1. Install [MongoDB CE v5.0.6](https://www.mongodb.com/try/download/community).
\ No newline at end of file
+1. Install [MongoDB CE v5.0.6](https://www.mongodb.com/try/download/community).
+
+### For WSL users:
+1. Follow [these instructions](https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-database#install-mongodb) to install MongoDB.
```

Comment:
Do we start the mongodb server by running: `sudo mongod --dbpath ~/data/db`? If so, can we include it in the doc so others (and yourself, haha) know how to start it?

Reply:
Would it be redundant to include it in the doc if it's mentioned as one of the steps in the link? Or do you want to include it more as a reminder for users to know how to run a Mongo instance anytime after they had first installed it?